server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -118,12 +119,14 @@ public URL getPropsUrl() {
     else if(!urlString.endsWith(".properties")) {
       urlString += (File.separatorChar + IoTDBConfig.CONFIG_NAME);
     }
-    // If the url doesn't start with "file:" it's provided as a normal path.
-    // So we need to add it to make it a real URL.
-    if(!urlString.startsWith("file:")) {
-      urlString = "file:" + urlString;
-    }
+
+    // If the url doesn't start with "file:" or other protocols, it's provided as a normal path.
+    // If valid, we return a URL converted from this file path.
+    File confFile = new File(urlString);
     try {
+      if (confFile.exists()) {
+        return confFile.toURI().toURL();
+      }
```

Comment:
This change will again break the ability to load a config properites resource from classpath ...